code/game/objects/items/weapons/weaponry.dm

```diff
@@ -72,6 +67,13 @@
 			user << "<span class='danger'>The rod appears to do nothing.</span>"
 			M.visible_message("<span class='danger'>\The [user] waves \the [src] over \the [M]'s head.</span>")
 			return
+		M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Is being deconverted with the [src.name] by [user.name] ([user.ckey])</font>")
+		user.attack_log += text("\[[time_stamp()]\] <font color='red'>Used the [src.name] to attempt to deconvert [M.name] ([M.ckey])</font>")
+
+		msg_admin_attack("[user.name] ([user.ckey]) attempted to deconvert [M.name] ([M.ckey]) with [src.name] (INTENT: [uppertext(user.a_intent)]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)")

```

Comment:
There exist helper methods for this. Use `[key_name(user)]` instead of `[user.name] ([user.ckey])`. It gives admins more info (like a PM link, shows if antag, etcetera. And `[key_name(M)]`.